src/presentation/services/rundown-event-service.ts

```diff
@@ -114,6 +111,10 @@ export class RundownEventService implements RundownEventEmitter, RundownEventObs
     const event: SegmentDeletedEvent = this.rundownEventBuilder.buildSegmentDeletedEvent(rundown, segmentId)
     this.emitRundownEvent(event)
   }
+  public emitSegmentUnsynced(rundown: Rundown, unsyncedSegment: Segment, originalSegmentId: string): void {
```

Comment:
A newline between methods is greatly appreciated.
Goes for the two other methods in this file that are also missing a new line.

Reply:
I think someone else fell into the webstorm-"looks like there is a spaces but its just information above the method"-issue alongside me, almost the whole file had this issue :-)